jsonschema/tests/test_validators.py

```diff
@@ -978,9 +989,9 @@ def test_it_returns_true_for_formats_it_does_not_know_about(self):
         )
         validator.validate("bugs")
 
-    def test_it_does_not_validate_formats_by_default(self):
+    def test_it_validates_formats_by_default(self):
         validator = self.Validator({})
-        self.assertIsNone(validator.format_checker)
+        self.assertIsInstance(validator.format_checker, FormatChecker)
```

Comment:
This test shouldn't pass, so if it does, something seems wrong -- the goal here wasn't to change the default for all validators, it was just to do so for calling `.check_schema`